plugins/usepe_logger.py

```diff
@@ -106,6 +115,17 @@ def los_logger(self):
         # Store the new loss of separation environment
         self.prevlos = currentlos
 
+    def cpa_logger(self):
+        ''' Logs CPA for each conflict. '''
+        loggedpair = list()
+
+        # For the best estimate, CPA is logged just after it happens (when tcpa turns negative)
+        for x in range(len(traf.cd.confpairs)):
+            pair = sorted(traf.cd.confpairs[x])
+            if (-1 < traf.cd.tcpa[x] <= 0) and (pair not in loggedpair):
```

Comment:
This -1 value is related to the update_interval? If so, I would change this to be equal to (or larger than) the update interval. Otherwise, the function may not log the cpa for update intervals larger than 1, if I understood this properly.

Reply:
This shall actually be -1 * the update interval. I added a commit to make this more clear. This is needed to only catch each CPA once.